manifest.webapp

```diff
@@ -69,6 +69,15 @@
         "ALL"
       ]
     },
+    "contacts": {
+      "description": "Required for the service to update the myself contact",
+      "type": "io.cozy.contacts",
+      "selector": "me",
```

Comment:
FMI, what is that for?

Reply:
This gives permission tu create or update the "myself" contact (and only it)